### PR TITLE
update car_cost_tool

### PR DESCRIPTION
**Pull request by paulanthonywilson**
Updates to version with ability to have a failover password

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -353,6 +353,8 @@ GEM
       railties (>= 3.0)
     html_validation (1.1.2)
     htmlentities (4.3.3)
+    http-cookie (1.0.2)
```
Comment (paulanthonywilson) [inline Gemfile.lock]: used by rest-client

File: Gemfile.lock [hunk 2]
```diff
@@ -506,7 +508,8 @@ GEM
     request_store (1.0.8)
     responders (1.1.2)
       railties (>= 3.2, < 4.2)
-    rest-client (1.7.3)
+    rest-client (1.8.0)
```
Comment (paulanthonywilson) [inline Gemfile.lock]: used by car_cost_tool

File: Gemfile.lock [hunk 3]
```diff
@@ -514,10 +517,13 @@ GEM
       css_parser (~> 1.3.4)
       nokogiri (> 1.5.0)
       sprockets
-    roo (1.13.2)
+    roo (2.0.0)
+      nokogiri (~> 1)
+      rubyzip (~> 1.1, < 2.0.0)
+    roo-xls (1.0.0)
```
Comment (paulanthonywilson) [inline Gemfile.lock]: Used by fuel price scraper in car cost tool

**Comment by darokel** [comment]
:+1:

**Comment by neoeno** [comment]
:+1:

**Comment by mconnell** [comment]
:+1:

**Pull request merged**